pages/blog/tooling-update-1.md

```diff
@@ -0,0 +1,75 @@
+- [CameronRushton](https://github.com/CameronRushton) [added a missing import](https://github.com/asyncapi/java-spring-cloud-stream-template/pull/244).
+- [CameronRushton](https://github.com/CameronRushton) [fixed generation failed when payload schemas had an `$id` field](https://github.com/asyncapi/java-spring-cloud-stream-template/pull/241).
+
+### Glee
+- [sudoshreyansh](https://github.com/sudoshreyansh) [when Glee detect invalid returned data it is now logged](https://github.com/asyncapi/glee/pull/273)
+- [sudoshreyansh](https://github.com/sudoshreyansh) [added a bunch of more missing tests](https://github.com/asyncapi/glee/pull/266)
+
+### Optimizer
+- [KhudaDad414](https://github.com/KhudaDad414) [refactored the parsing process to easer add new paths used for optimization](https://github.com/asyncapi/optimizer/pull/85)
+
+### Go Watermill template
+- [anandsunderraman](https://github.com/anandsunderraman) [added support for AMQP publisher](https://github.com/asyncapi/go-watermill-template/pull/137)
+
+### Modelina
+- [Samridhi-98](https://github.com/Samridhi-98) [fixed Java preset options was not added as type](https://github.com/asyncapi/modelina/pull/716)
+- [Samridhi-98](https://github.com/Samridhi-98) [fixed Javascript preset options was not added as type](https://github.com/asyncapi/modelina/pull/709)
+- [Samridhi-98](https://github.com/Samridhi-98) [fixed TypeScript preset options was not added as type](https://github.com/asyncapi/modelina/pull/703)
+- [Samridhi-98](https://github.com/Samridhi-98) [fixed some broken links in the documentation](https://github.com/asyncapi/modelina/pull/684)
+- [Samridhi-98](https://github.com/Samridhi-98) [ensured that reserved keywords cannot be rendered for Go](https://github.com/asyncapi/modelina/pull/653)
+- [slowikowskiarkadiusz](https://github.com/slowikowskiarkadiusz) [added support for shorthand accessors for C# classes](https://github.com/asyncapi/modelina/pull/671)
+- [slowikowskiarkadiusz](https://github.com/slowikowskiarkadiusz) [added support for different collection types for C# list properties](https://github.com/asyncapi/modelina/pull/669)
+- [ibernabeudev](https://github.com/ibernabeudev) [added enum rendering to Go](https://github.com/asyncapi/modelina/pull/662)
+- [jonaslagoni](https://github.com/jonaslagoni) [added new core model and started the refactoring](https://github.com/asyncapi/modelina/pull/655), see more on the [`next` branch](https://github.com/asyncapi/modelina/tree/next)
+
+### Simulator
+- [NektariosFifes](https://github.com/NektariosFifes) [added the initial setup for the electron application](https://github.com/asyncapi/simulator/pull/103), that will serve as a dashboard for simulation 
+
+### Generator
+- [magicmatatjahu](https://github.com/magicmatatjahu) [fixed ts-node would use multiple instances](https://github.com/asyncapi/generator/pull/764)
+
```

Comment:
Since this is a blog post, do you think you want to add a closing statement or goodbye? 😸 

It seems abrupt to not have a short closing to the blog post for the reader.

Reply:
I think you are right yea 🤔 

I cant really think of anything to add, any suggestions?

Reply:
```suggestion
## In closing
I hope you enjoyed my first AsyncAPI Tooling blog update, and I look forward to your feedback. 

See you in the next Tools update!
```

Sure, maybe something similar to this? 😄✌🏽